scripts/globals/follow.lua

```diff
@@ -6,144 +6,19 @@ require('scripts/globals/utils')
-        local mobPos = mob:getPos()
-
-        if
-            mob:isSpawned() and
-            not utils.distanceWithin(leaderPos, mobPos, withinDistance, true) and
-            utils.angleWithin(leaderPos, followerPos, mobPos, withinAngle)
-        then
-            table.insert(filteredPos, mobPos)
-        end
-    end
-
-    return filteredPos
-end
-
-local function onMobRoamAction(follower)
-    local leader = xi.follow.getFollowTarget(follower)
-
-    if
-        not leader or
-        not leader:isSpawned() or
-        leader:getZoneID() ~= follower:getZoneID()
-    then
-        xi.follow.stopFollowing(follower)
-        return
-    end
-
-    if follower:isFollowingPath() then
-        return
-    end
-
-    local followerId = follower:getID()
-    local options    = followerOptions[followerId]
-
-    if options.forceRepathInterval then
-        local lastPath = follower:getLocalVar('[follow]lastPath') + 1
-        follower:setLocalVar('[follow]lastPath', lastPath)
-        follower:queue(options.forceRepathInterval * 1000, function()
-            if follower:getLocalVar('[follow]lastPath') == lastPath then
-                follower:clearPath()
-            end
-        end)
-    end
-
-    local leaderId         = leader:getID()
-    local leaderPos        = leader:getPos()
-    local followerPos      = follower:getPos()
-    local followerDistance = utils.distance(leaderPos, followerPos)
-    local followDistance   = options.followDistance
-    local separationAngle  = options.separationAngle
-
-    if options.warpDistance and followerDistance > options.warpDistance then
-        follower:teleport(utils.getNearPosition(leaderPos, 2 + math.random() * (followDistance - 2), math.random() * 2 * math.pi), leader)
-    elseif followerDistance > followDistance + 1.0 then
-        local followers     = leaderToFollowersMap[leaderId]
-        local followerCount = #followers
-        local followerIndex
-
-        for i, mob in ipairs(followers) do
-            if mob:getID() == followerId then
-                followerIndex = i
-                break
-            end
-        end
-
-        local filteredFollowersPos = filterFollowersToPosArray(leaderPos, followerPos, followers, followDistance, separationAngle)
-        local targetAngle
-
-        if #filteredFollowersPos == 1 then
-            -- Ignore other followers, go straight to leader.
-            targetAngle = utils.getWorldAngle(leaderPos, followerPos)
-        else
-            -- Go to leader while leaving room for other followers.
-            targetAngle = getAverageWorldAngle(leaderPos, filteredFollowersPos)
-            targetAngle = targetAngle + (separationAngle * (followerIndex - 1)) - (separationAngle * (followerCount - 1) / 2)
-        end
-
-        local targetPos = utils.getNearPosition(leaderPos, followDistance, targetAngle)
-        follower:pathTo(targetPos.x, targetPos.y, targetPos.z, options.pathFlags)
-    end
-end
+xi.follower = xi.follower or {}
```

Comment:
This now no longer lines up with the file that contains is follow / follower etc.